website/content/docs/concepts/stateful-deployments.mdx

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+page_title: Stateful Deployments
+description: |-
+  Learn how Nomad handles Stateful Deployments.
```

Comment:
```suggestion
  Learn how Nomad handles stateful deployments. Use a dynamic host volume for your stateful workload and bind an allocation to a volume ID with the sticky parameter. Learn how Nomad scales jobs with sticky volumes. 
```

Updating the description to include more of what is on the page because developer.hashicorp.com indexes the front matter description for the site search. The local site search does not index page contents.